rpc/jsonrpc/server/http_json_handler.go

```diff
@@ -98,7 +98,7 @@ func makeJSONRPCHandler(funcMap map[string]*RPCFunc, logger log.Logger) http.Han
 				args = append(args, fnArgs...)
 			}
 			returns := rpcFunc.f.Call(args)
-			logger.Info("HTTPJSONRPC", "method", request.Method, "args", args, "returns", returns)
```

Comment:
why not debug?

Reply:
i can change it

Reply:
https://github.com/tendermint/tendermint/pull/6198/commits/8d939e99a7e4ca47103c4e6a59e5c8637c1b8a62

Reply:
Didn't bez already remove this in his PR #6201 ?

Reply:
yea, this would revert it to a debug. Dont have a preference which direction we go.